A patch that will be applied to presence.nvim on Windows installations.
See https://github.com/andweeb/presence.nvim/pull/87.

diff --git a/lua/presence/init.lua b/lua/presence/init.lua
index 1d2aefe..ce7b067 100644
--- a/lua/presence/init.lua
+++ b/lua/presence/init.lua
@@ -768,6 +768,11 @@ end
 
 -- Update Rich Presence for the provided vim buffer
 function Presence:update_for_buffer(buffer, should_debounce)
+    -- Since git always uses forward slashes, replace with backslash in Windows
+    if self.os.name == "windows" then
+        buffer = buffer:gsub("/", [[\]])
+    end
+
     -- Avoid unnecessary updates if the previous activity was for the current buffer
     -- (allow same-buffer updates when line numbers are enabled)
     if self.options.enable_line_number == 0 and self.last_activity.file == buffer then
